README.md

```diff
@@ -29,7 +31,14 @@ Instantiate an instructor named 'Robin' and call its `givesHomework` method with
 Your Answer:
 
 ```js
-// your code here
+function Instructor(name, assignment) { // name property, giveHomework Method with assignment argument. When executed console.log
+  this.name = name;
+  this.assignment = assignment;
+  givesHomework = function() {
```

Comment:
We were expecting assignment to be passed to "giveHomework".  The Instructor doesn't own the assignment.  He gives many assignments as homework.